contributors/devel/cri-validation.md

```diff
@@ -0,0 +1,50 @@
+# CRI Validation Testing
+
+CRI validation testing provides a series of common validation test suites for Kubelet CRI (_Container Runtime Interface_).
```

Comment:
"provides a test framework and a suite of tests to validate that the Container Runtime Interface (CRI) server implementation meets all the requirements. This allows the CRI runtime developers to verify that their runtime conforms to CRI, without needing to set up Kubernetes components or run Kubernetes end-to-end tests."